R/parse_obs_types.R

```diff
@@ -0,0 +1,25 @@
+#' Parse observation types to simulation code
+#'
+#' @param obs specified observation object, e.g. `list(variable="CONC", scale="1")` or `list(cmt=1, scale="V/1000.0")`.
```

Comment:
is the second example valid here? This function will only get called if the following is true:

```
!is.null(obs$variable) && length(obs$variable) > 0
```
Yet the second example is missing a `variable` object. I suggest clarifying the expectations in the documentation.

```suggestion
#' @param obs specified observation object including at least a description of 
       which variable(s) are associated with a particular compartment, e.g. 
       `list(variable="CONC", scale="1")`.
```